tests/test_frame.py

```diff
@@ -1915,3 +1915,9 @@ def f(s: pd.DataFrame) -> pd.Series:
         return s.mean()
 
     check(assert_type(resampler.apply(f), Union[pd.Series, pd.DataFrame]), pd.DataFrame)
+
+
+def df_accepting_dicts_iterator() -> None: 
+    # GH 392
+    data = [{"a": 1, "b": 2}, {"a": 3, "b": 5}]
+    p = pd.DataFrame(iter(data))
```

Comment:
One more thing.  Can you change this last line to:
```python
    check(assert_type(pd.DataFrame(iter(data)), pd.DataFrame), pd.DataFrame)
```

Reply:
Sure

Reply:
Done